addons/web_editor/static/src/js/common/utils.js

```diff
@@ -319,7 +319,12 @@ function _getCSSVariableValue(key, htmlStyle) {
     // quoted for strings or non quoted for colors, numbers, etc. However,
     // Chrome has the annoying behavior of changing the single-quotes to
     // double-quotes when reading them through getPropertyValue...
-    return value.replace(/"/g, "'");
+    value = value.replace(/"/g, "'");
+    // Prevent Python boolean style and keep safe retro-compatibility
+    if (["'True'", "'False'"].includes(value)) {
+        value = value.replace(/'/g, "").toLowerCase()
```

Comment:
You need to install a linter, otherwise we will have to comment every line of your PR because of a missing space or something like that :wink: Here missing the trailing `;`. There is the official one described here https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines, although I don't think many use it in the website team (it might not even be enabled for website files). Ask your friendly neighbors what they are using if your linter is not set up yet :+1: 

I guess that when we'll reach 100% OWL convertion we'll align ourselves with the same linter as the framework team. A discussion for after OXP :)